google-cloud-pubsublite/pom.xml

```diff
@@ -195,10 +183,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>flatten-maven-plugin</artifactId>
-      </plugin>
```

Comment:
Pinging our flatten maven plugin expert @stephaniewang526 here. Does flatten maven plugin conflict with errorprone?

Reply:
No, it shouldn't. I would not recommend removing it.

Reply:
This isn't removed, just moved to the plugins section on the parent POM

Reply:
Okay as long as you're okay with generating flattened poms for parent/root poms as well; this is fine.